hc/api/tests/test_list_checks.py

```diff
@@ -30,19 +31,23 @@ def setUp(self):
 
     def get(self):
         return self.client.get("/api/v1/checks/", HTTP_X_API_KEY="abc")
-
+    
     def test_it_works(self):
         r = self.get()
+        self.assertEqual(r.status_code, 200)
         ### Assert the response status code
 
         doc = r.json()
         self.assertTrue("checks" in doc)
 
         checks = {check["name"]: check for check in doc["checks"]}
+        self.assertTrue(len(checks) == 2)
+        self.assertTrue(("timeout" and "grace" and "ping_url" and "status" and "last_ping" and "n_pings" and "pause_url") in checks["Alice 1"].keys())
```

Comment:
this seems like a lot of joined statements. It should be broken down to be more readable